src/main/java/de/qabel/desktop/ui/actionlog/item/renderer/PlaintextMessageRenderer.java

```diff
@@ -2,20 +2,73 @@
 
 import de.qabel.desktop.daemon.drop.TextMessage;
 import javafx.scene.Node;
+import javafx.scene.control.Hyperlink;
 import javafx.scene.control.Label;
+import javafx.scene.text.TextFlow;
+import org.controlsfx.control.HyperlinkLabel;
+import org.jetbrains.annotations.NotNull;
 
+import java.awt.*;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
 import java.util.ResourceBundle;
+import java.util.function.Consumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class PlaintextMessageRenderer implements FXMessageRenderer {
+    private static final String STYLE_CLASS = "message-text";
+    public Consumer<String> browserOpener;
+
+    public PlaintextMessageRenderer() {
+        browserOpener = (uri) -> {
```

Comment:
You can also initialize it inline where you declare the member. Functionally it is identical.

A better way do to this is imho to create a dedicated browser opener interactor and replace the real one, which has Desktop.getDesktop() stuff, with a testable browser opener.